graftm/hmmer.py

```diff
@@ -507,56 +500,57 @@ def alignment_correcter(self, alignment_file_list, output_file_name):
                         output_file.write(fasta_id)
                         output_file.write(fasta_seq)
 
-    def extract_orfs(self, input_path, raw_orf_path, hmmsearch_out_path, orf_titles_path, orfm, orf_out_path):
+    def extract_orfs(self, input_path, orfm, orf_out_path):
         '''
-        orfm: graftm.OrfM object with parameters already set
+        extract_orfs - Extract only the orfs that hit the hmm, return sequence 
+                       file with within.
```

Comment:
How about "Return a path to a FASTA file containing ORFs that hit the HMM (self.search_hmm)